src/genie/libs/parser/ios/asr901/tests/ShowEnvironment/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,72 @@
+expected_output = {
+    "Fan": {
+        "fan1": {
```

Comment:
there is key name `fan`. so can have only number under the key.
```suggestion
    "fan": {
        1: {
```